```console
/home/swiftci/jenkins/workspace/swift-PR-TensorFlow-Linux/swift/stdlib/public/Python/Python.swift:227:25: error: use of undeclared type 'LocalizedError'
extension PythonError : LocalizedError {
                        ^~~~~~~~~~~~~~
```

Hmm. Not sure why `LocalizedError` doesn't exist here.

--- a/stdlib/public/Python/Python.swift
+++ b/stdlib/public/Python/Python.swift
@@ -224,12 +224,6 @@ extension PythonError : CustomStringConvertible {
   }
 }
 
-extension PythonError : LocalizedError {
-  public var errorDescription: String? {
-    return self.description
-  }
-}
-
 // Reflect a Python error (which must be active) into a Swift error if one is
 // active.
 private func throwPythonErrorIfPresent() throws {

